File: 01_OOPS-Concepts/python/ClassObjects/01_Student_data.py
class Student:
    def __init__(self, name, age, marks):
        self.name = name
        self.age = age
        self.marks = marks
        
    # def get_details(self):
    #     return "name is {0}, age is {1} and {2} marks scored".format(self.name, self.age, self.marks)
    
    def get_details(self):
        return f"name is {self.name}, age is {self.age} and {self.marks} scored"
    
std1 = Student('Joshi', 25, 85)
print(std1.get_details())

File: 01_OOPS-Concepts/JavaScript/ClassObjects/01_StudentData.js
class Student{
    name
    age
    marks

    constructor(name, age, marks){
        this.name = name;
        this.age = age;
        this.marks = marks
    }

    getDetails(){
        return `Name is ${this.name}, age is ${this.age} and ${this.marks} marks scored`
    }
}

const std = new Student("Joshi", 25, 85)
console.log(std.getDetails());
